src/org/opendatakit/briefcase/ui/reused/DetailsStatusButton.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2018 Nafundi
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.opendatakit.briefcase.ui.reused;
+
+import static java.awt.Color.DARK_GRAY;
+import static java.awt.Color.LIGHT_GRAY;
+
+import java.awt.Font;
+import java.awt.Insets;
+import javax.swing.JButton;
+
+public class DetailsStatusButton implements ButtonProcessing, Comparable<ButtonProcessing> {
+  private static final Font IC_RECEIPT = FontUtils.getCustomFont("ic_receipt.ttf", 16f);
+
+  private boolean status = false;
+  private final JButton button;
+
+  private DetailsStatusButton(JButton button) {
+    this.button = button;
+  }
+
+  @SuppressWarnings("checkstyle:AvoidEscapedUnicodeCharacters")
+  public static DetailsStatusButton create() {
+    JButton button = new JButton();
+    button.setText("\uE900");
+    button.setFont(IC_RECEIPT);
+    button.setToolTipText("View this form's status history");
+    button.setMargin(new Insets(0, 0, 0, 0));
+    button.setForeground(LIGHT_GRAY);
+    return new DetailsStatusButton(button);
+  }
+
+  public JButton getJButton() {
+    return button;
+  }
+
+  public void setStatus (boolean value){
+    status = value;
+    button.setForeground(status ?  DARK_GRAY : LIGHT_GRAY );
+  }
+
+  public int compareTo(ButtonProcessing b) {
```

Comment:
So, this method should be migrated to `public static int compare(DetailsStatusButton a, DetailsStatusButton b)`

Reply:
See earlier comment on Comparable